src/hooks/useInitializeCollectionDraft.ts

```diff
@@ -107,20 +111,27 @@ function useInitializeCollectionDraft() {
                     spec: newDraftSpecResponse.data[0].spec,
                     belongsToDraft: true,
                 });
+
+                setCollectionInitializationDone(true);
             } else {
                 updateBindingsEditorState({
                     spec: liveSpec,
                     belongsToDraft: false,
                 });
 
+                setCollectionInitializationDone(false);
```

Comment:
Considered just setting this from within the alert setting but originally I was setting this to `true` even when we got an alert and nothing bad happened.

Gonna leave it here to give more flexibility.